src/lib/php/Dao/SysConfigDao.php

```diff
@@ -62,4 +66,18 @@ public function getCustomiseData($data)
     }
     return $finalVal;
   }
+
+  /**
+   * \brief Update Configuration Data.
+   *  @return Info
+   */
+  public function UpdateConfigData($data)
+  {
+    $key = strval($data['key']);
+    $value = strval($data['value']);
+    $stmt = __METHOD__ . 'UpdateConfigData'.$key;
+    $sql = "UPDATE sysconfig SET conf_value = $2 WHERE variablename = $1";
+    $this->dbManager->getSingleRow($sql, array($key, $value), $stmt);
+    return (new Info(200, 'Succesfully Updated Customise data', InfoType::INFO));
```

Comment:
Library functions should not use classes from other name space. Please return boolean `true` or `false.`